SR2021/2020-09-25-state-of-affairs.md

```diff
@@ -0,0 +1,22 @@
+---
+to: All Teams
+subject: Student Robotics Update
+---
+
+Hi All,
+
+Just a quick update on the current state of affairs with Student Robotics.
+
+## Student Robotics 2021
+
+Due to recent events, and the current measures in place, We will not be running a **physical** competition for Student Robotics 2021.
```

Comment:
What does this mean?  Does it mean that there will be no robots, or does it mean that people will not be attending in person?

Reply:
Mostly the latter, although it's left intentionally vague to allow the former to be a possibility, should it be decided.